updated Jenkins on AWS tutorial documentation (#5629)
jenkins-infra/jenkins.io: content/doc/tutorials/tutorial-for-installing-jenkins-on-AWS.adoc

. An *AWS account*. If you don't have one, you can register link:https://portal.aws.amazon.com/billing/signup#/start[here].
. An Amazon EC2 key pair. If you don't have one, refer to <<Creating a key pair>>.

> . An AWS IAM User with programmatic key access and permissions to launch EC2 instances

Reviewer: ```suggestion
. An AWS IAM User with programmatic key access and link:https://plugins.jenkins.io/ec2/#plugin-content-iam-setup[permissions to launch EC2 instances]
```

What do you think about linking to the documentation chapter which lists the required IAM permissions (as it's not only `ec2:CreateVM` which is required)?

+
image::tutorials/AWS/ec2_choose_ami.png[Choosing an Amazon Machine Image]

> . Scroll down and select select the key pair you created in the <<Creating a key pair, creating a key pair>> section above or any existing key pair you intend to use.

Reviewer 2: ```suggestion
. Scroll down and select the key pair you created in the <<Creating a key pair, creating a key pair>> section above or any existing key pair you intend to use.
```

Under Network settings:
.. Select *Select an existing security group*.
.. Select the *WebServerSG* security group that you created.
.. Select *Launch Instance*.
+

> image::tutorials/AWS/ec2_review_instance_launch.png[Review your amazon launch instance]

Reviewer 2: ```suggestion
image::tutorials/AWS/ec2_review_instance_launch.png[Review your Amazon launch instance]
```

+
image::tutorials/AWS/add-amazon-cloud.png[Adding Amazon EC2 cloud to Jenkins.]

. Click Add under Amazon EC2 Credentials
+
image::tutorials/AWS/configure_cloud_add_ec2_credentials.png[Adding EC2 credentials in Configure Cloud.]
.. From the Jenkins Credentials Provider, choose AWS Credentials as the Kind
+
image::tutorials/AWS/jenkins_credentials_provider_aws_credentials.png[Choosing Kind AWS Credentials.]
.. Scroll down and enter in the IAM User programmatic access keys with permissions to launch EC2 instances and click Add
+
image::tutorials/AWS/add_access_secret_access_keys.png[Adding AWS Credentials.]

> .. Scroll down to select your region using the drop-down, and click Add for the EC2 Key Pair's Private Key

Reviewer 3: ```suggestion
.. Scroll down to select your region using the drop-down, and select *Add* for the EC2 Key Pair's Private Key.
```
small style and syntax update

+
image::tutorials/AWS/configure_cloud_region_private_key.png[Set Region and add Private Key.]

> .. From the Jenkins Credentials Provider, choose SSH Username with private key as the Kind and set the Username to ec2-user

Reviewer 3: ```suggestion
.. From the Jenkins Credentials Provider, select SSH Username with private key as the Kind and set the Username to `ec2-user`.
```
small syntax and format suggestion

+
image::tutorials/AWS/ssh_username.png[Set Kind to SSH Username with private key.]

> .. Scroll down and choose Enter Directly under Private Key and click Add

Reviewer 3: ```suggestion
.. Scroll down and select *Enter Directly* under Private Key, then select Add.
```
small wording/formatting suggestion

+
image::tutorials/AWS/private_key_enter_directly.png[Set Private Key to Enter Directly.]

> .. Open the private key pair you created in the <<Creating a key pair, creating a key pair>> directly and paste in the contents from "-----BEGIN RSA PRIVATE KEY-----" to "-----END RSA PRIVATE KEY-----". Click Add when completed

Reviewer 3: ```suggestion
.. Open the private key pair you created in the <<Creating a key pair, creating a key pair>> step and paste in the contents from "-----BEGIN RSA PRIVATE KEY-----" to "-----END RSA PRIVATE KEY-----".
Select Add when completed.
```
small syntax and formatting suggestion, just to make sure the instructions are deliberate in what they say

+
image::tutorials/AWS/enter_private_key.png[Enter Private Key.]

> .. Scroll down to "Test Connection" and ensure it states "Success". Hit Save when done

Reviewer 3: ```suggestion
.. Scroll down to "Test Connection" and ensure it states "Success".
Select *Save* when done
```
small formatting and syntax suggestion for consistency

+
image::tutorials/AWS/add-amazon-cloud.png[Adding Amazon EC2 cloud to Jenkins.]

> . Click Add under Amazon EC2 Credentials

Reviewer 2: ```suggestion
. Click *Add* under Amazon EC2 Credentials
```
